recipes/openscenegraph/all/conanfile.py

```diff
@@ -192,7 +189,6 @@ def _configured_cmake(self):
         cmake.definitions["OSG_WITH_INVENTOR"] = False
         cmake.definitions["OSG_WITH_JASPER"] = self.options.with_jasper
         cmake.definitions["OSG_WITH_OPENCASCADE"] = False
-        cmake.definitions["OSG_WITH_COLLADA"] = False  # self.options.with_collada
```

Comment:
Why are you removing these (commented out) options?

Reply:
So there are a whole bunch of plugins that haven't been implemented (see package_info), and it isn't clear to me how all these options should be named. E.g. Should they be off of the plugin name or the dependency name? I didn't really think about any of the plugins that didn't have the necessary dependencies already available on CCI when writing this recipe, so I figured it would be just better to clean up not thought through stuff.

Reply:
I suggest you leave you them or add a fix me so we know they are missing (the idea is to minimize the changes in package ID when new ones get added)

Reply:
I understand that, but that only applies if the option name is decided upon, which was one of my questions. Also, there are at least 14 unhandled plugins at the moment, and I don't intend to address them all in this PR, so that will inevitably mean the package ids will change in the future. I think asio is fair since it is essentially handled, it is just waiting on upstream bug fixes.